### DISCUSSION TITLE
Prijava projekta (PI21-24)

### DISCUSSION THREAD

**Discussion by zstapic** [foivz/pi21-vtucelj20-icuturic20-ipeles#1] [General]
Poštovane kolegice,

oznaka vašeg tima je PI21-24, a vaš mentor sam ja.

Tema projekta koju ste prijavili u skladu je sa očekivanjima na kolegiju, a popis funkcionalnosti molim da kratko dopunite prema sljedećim uputama:

- proizvod dizajnirati na način da ga možete prodati/instalirati na više škola. Potrebna vam je uloga super admina koji može kreirati školu i njezinog administratora koji onda provodi F07. Svakom dodanom korisniku (administratoru ili nastavniku ili učeniku) poslati mail sa pristupnim podacima u sustav (Peleš)
- dopuniti F03 da da učenik i nastavnik nakon učeničkog rješavanja kviza dobije na mail PDF sa svim pitanjima koja je učenik imao i sa njegovim odgovorima i rezultatom (Tucelj) 
- dopuniti F08 sa tablicama uz grafove te sa mogućnošću izvoza u PDF (Čuturić)

Molim da sukladno uputama dopunite prijavu i da započnete raditi na projektu.

Predlažem vam, iako nije obvezno, da umjesto MySQL koristite MSSQL koji ćemo koristiti na nastavi i za koji vam nastavnici budu dali pristup bazi. Za MySQL nemamo podršku na kolegiju.

Molim također samo kratki odgovor ovdje da ste postupili po uputama da znam da ste ih pročitali. Ako imate bilo kakvih pitanja, slobodno ih ovdje postavite.

**Comment by irensi** [comment]
Poštovani profesore, 
Pročitale smo Vaše upute i promijenile popis funkcionalnosti prema Vašim uputama.

**Comment by zstapic** [comment]
Super. Provjerit ću dok stignem. Vi samo nastavite raditi na projektnom
planu i detaljnoj specifikaciji zahtjeva!
